web/src/lib/components/asset-viewer/photo-sphere-viewer-adapter.svelte

```diff
@@ -28,6 +30,18 @@
       maxFov: 180,
       fisheye: true,
     });
+
+    if (originalImageUrl) {
+      const zoomHandler = ({ zoomLevel }: events.ZoomUpdatedEvent) => {
+        // zoomLevel range: [0, 100]
+        if (Math.round(zoomLevel) >= 75) {
+          // Replace the preview with the original
+          viewer.setPanorama(originalImageUrl, { showLoader: false, speed: 150 }).catch(() => {});
```

Comment:
If the original image is unsupported a `The panorama cannot be loaded` overlay will show up. I think we should do one or both of the following:
- Only load the image when `isWebCompatibleImage(asset)` is true
- Fall back to the previous image when loading fails

Reply:
Added `isWebCompatibleImage` check to set the `originalImageUrl`, and changed the catch for `setPanorama` to set it back to the preview image.